### PR TITLE
test(manual) update Single replica node down

### PR DESCRIPTION
**Pull request by chriscchien**
#### Which issue(s) this PR fixes:

Issue  longhorn/longhorn#8350

#### What this PR does / why we need it:

Add test scenario for volume and the only replica not on the same node.

#### Special notes for your reviewer:

N/A

#### Additional documentation or context

N/A

### CODE REVIEW

File: docs/content/manual/pre-release/node-not-ready/node-down/single-replica-node-down.md [hunk 1]
```diff
@@ -11,26 +11,46 @@ https://github.com/longhorn/longhorn/issues/3957
...
+3. Power down node of volume.
 4. Volume will become faulted.
 5. Wait for pod deletion & recreation on another node.
    The pod recreation will not happen immediately.
 6. The replacement workload pod will get stuck in the `ContainerCreating` state.
-7. Power on node of volume/replica.
-8. Verify the auto salvage finishes for volumes.
-9. Verify volume attached & accessible by pod (i.e test data is available).
\ No newline at end of file
+7. Power on the off node.
```
Comment (khushboo-rancher) [inline docs/content/manual/pre-release/node-not-ready/node-down/single-replica-node-down.md:45]: Typo: `Power on the node that is off` or `Power on the node`

**Review by khushboo-rancher — CHANGES_REQUESTED** [review]
@chriscchien Have we covered all the scenarios in the automation?

**Comment by chriscchien** [comment]
> @chriscchien Have we covered all the scenarios in the automation?

Updated and also update some type found by bot.
For the scenarios, yes I think automation covered all the scenarios.

**Comment by khushboo-rancher** [comment]
>For the scenarios, yes I think automation covered all the scenarios.

It would be fine to remove the cases/file from manual test then :)

**Comment by chriscchien** [comment]
> > For the scenarios, yes I think automation covered all the scenarios.
> 
> It would be fine to remove the cases/file from manual test then :)

Updated!

**Review by khushboo-rancher — APPROVED** [review]

**Pull request merged**